.github/workflows/verify.yml

```
@@ -104,7 +104,7 @@ jobs:
         run: |
           MSG=$(git log --format=%B -n 1 ${{github.event.after}})
           echo COMMIT_MESSAGE=\"$MSG\" >> $GITHUB_ENV
-      - name: test
+      - name: Publish
         run: .github/scripts/publish_chart.sh
         shell: bash
-        if: github.ref == 'refs/heads/main' || env.COMMIT_MESSAGE == '[publish]'
+        if: github.ref_name == '${{ github.event.repository.default_branch }}' || env.COMMIT_MESSAGE == '[publish]'
```

```suggestion
        if: github.ref_name == github.event.repository.default_branch || env.COMMIT_MESSAGE == '[publish]'
```

it's not very intuitive, but `if` are implicitly an expression and not a string, see https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idif